packages/typespec-vscode/src/typespec-utils.ts

```diff
@@ -53,3 +55,27 @@ export async function TraverseMainTspFileInWorkspace() {
         .map((uri) => normalizeSlashes(uri.fsPath)),
     );
 }
+
+export function getVscodeUriFromPath(path: string) {
```

Comment:
Feels not that necessary considering it's just one line code. But if you do think it's useful, please move it to util instead of typespec-util which is not related to typespec.